linkcheck/plugins/markdowncheck.py

```diff
@@ -125,9 +139,9 @@ def _find_balanced(self, text, start, open_c, close_c):
         end of string if it's reached before the balance point is found.
         """
         i = start
-        l = len(text)
+        n = len(text)
```

Comment:
uh! didn't know black cared about stuff like that, but i guess it makes sense because `l` could be ambiguous, in some fonts...

Reply:
That is a flake8 violation.